Sources/ApodiniObserve/LoggerExporter.swift

```diff
@@ -0,0 +1,86 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+// 
+
+import Apodini
+import ApodiniUtils
+
+/// An ``IntefaceExporter``that writes information from the ``Blackboard`` to the ``Environment`` of the ``Delegate``
+public final class LoggerExporter: InterfaceExporter, TruthAnchor {
+    /// Contains all the necessary information from the ``Blackboard``, then accessed via the ``Environment`` property wrapper of the ``ConfiguredLogger``
+    public struct BlackboardMetadata: EnvironmentAccessible {
+        public struct BlackboardMetadata {
+            let endpointName: String
+            let endpointParameters: EndpointParameters
+            let parameters: [ParameterRetriever]
+            let parameterTupels: [(String, ParameterRetriever)]
+            let operation: Apodini.Operation
+            let endpointPathComponents: EndpointPathComponents
+            let context: Context
+            let anyEndpointSource: AnyEndpointSource
+            let handleReturnType: HandleReturnType
+            let responseType: ResponseType
+            let serviceType: ServiceType
+        }
+        
+        public var value: BlackboardMetadata
+    }
+    
+    // Not sure if those are maybe needed?
+    let app: Apodini.Application
+    let exporterConfiguration: LoggerConfiguration
```

Comment:
Well, for now, they are not really needed. But all `Exporter`s are structured like that (application, exporterConfiguration) (maybe it would even make sense to include that in the `InterfaceExporter` protocol sometime), therefore I would leave the code in there.

Reply:
And, TBH, I just forgot to delete the comment :D

Reply:
I'm totally fine with leaving them in. The comment just seemed like it was forgotten 😁